apps/src/applab/designProperties.jsx

```diff
@@ -62,9 +62,9 @@ var DesignProperties = module.exports = React.createClass({
     var deleteButton;
     var element = this.props.element;
     // First screen is not deletable
-    var firstScreen = elementType === elementLibrary.ElementType.SCREEN &&
-        element.parentNode.firstChild === element;
-    if (!firstScreen) {
+    var onlyScreen = elementType === elementLibrary.ElementType.SCREEN &&
```

Comment:
nit: can you change this to `isOnlyScreen` ? We have a convention of doing this when possible for boolean values. Apologies in advance for all the counterexamples you will undoubtedly find to this rule.